include/lib/el3_runtime/cpu_data.h

```diff
@@ -25,14 +28,18 @@
 #endif /* AARCH32 */
 
 #if CRASH_REPORTING
-#define CPU_DATA_LOG2SIZE		7
 #define CPU_DATA_CRASH_BUF_END		(CPU_DATA_CRASH_BUF_OFFSET + \
 						CPU_DATA_CRASH_BUF_SIZE)
 #else
-#define CPU_DATA_LOG2SIZE		6
 #define CPU_DATA_CRASH_BUF_END		CPU_DATA_CRASH_BUF_OFFSET
 #endif
 
+/* cpu_data size is the data size rounded up to the platform cache line size */
+#define CPU_DATA_SIZE			(((CPU_DATA_CRASH_BUF_END + \
+					CACHE_WRITEBACK_GRANULE - 1) / \
+						CACHE_WRITEBACK_GRANULE) * \
+							CACHE_WRITEBACK_GRANULE)
```

Comment:
minor: could use `round_up()` in utils_def.h

Reply:
`round_up()` relies on `__typeof__` which is not supported in assembly.
Implement assembly compliant `round_up()/down()` in utils_def.h ?

Reply:
Hmm, good point. There's another patch in flight that adds an alternative round_up macro as you suggest. That doesn't need to delay this patch though (it can be fixed later). So I'll merge as-is.